ts/sagas/messages/watchThirdPartyMessageSaga.ts

```diff
@@ -0,0 +1,48 @@
+import { SagaIterator } from "redux-saga";
+import { ActionType, getType } from "typesafe-actions";
+import { call, put, takeLatest } from "typed-redux-saga/macro";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import { BackendClient } from "../../api/backend";
+import { loadThirdPartyMessage } from "../../features/messages/store/actions";
+import { getError } from "../../utils/errors";
+
+function* getThirdPartyMessage(
+  client: ReturnType<typeof BackendClient>,
+  action: ActionType<typeof loadThirdPartyMessage.request>
+) {
+  const id = action.payload;
+  try {
+    const result = yield* call(client.getThirdPartyMessage, { id });
+    if (result.isLeft()) {
+      yield* put(
+        loadThirdPartyMessage.failure({
+          id,
+          error: new Error(readableReport(result.value))
+        })
+      );
+    } else if (result.isRight() && result.value.status === 200) {
```

Comment:
I am quite sure you don't need to check "result.isRight()" because already asserted by the first if.

Reply:
Yup, I forgot to remove it when I added the `isLeft()` path above it. b36b30494d0813fc1f580eb96ef9fa1770317de7